packages/react-data-grid/src/ReactDataGrid.js

```diff
@@ -66,6 +66,8 @@ const ReactDataGrid = React.createClass({
     onCellsDragged: React.PropTypes.func,
     onAddFilter: React.PropTypes.func,
     onGridSort: React.PropTypes.func,
+    sortColumn: React.PropTypes.string,
+    sortDirection: React.PropTypes.oneOf(['ASC', 'DESC', 'NONE']),
```

Comment:
hey @tdzienniak -- this looks good, but one suggestion: can we use the `DEFINE_SORT` const from here

https://github.com/adazzle/react-data-grid/blob/master/packages/react-data-grid/src/cells/headerCells/SortableHeaderCell.js#L3

rather than having multiple arrays of possible values?

There's probably also an argument to be made for moving the existing array out of `SortableHeaderCell`.

My two cents are that `sortColumn` and `sortDirection` are in the right place here, but the existing definitions should be moved here too? Sorting should conceptually apply to the whole grid, not just HeaderCells..